src/map/clif.cpp

```diff
@@ -9814,7 +9774,7 @@ void clif_refresh(map_session_data *sd)
 	if( sd->state.vending )
 		clif_openvending( *sd );
 	if( pc_issit(sd) )
-		clif_sitting(&sd->bl); // FIXME: just send to self, not area
+		clif_sitting(sd->bl);
```

Comment:
```suggestion
		clif_sitting(sd->bl); // FIXME: just send to self, not area
```
Problem was not resolved. Please leave the FIXME comment.